pypeman/helpers/serializers.py

```diff
@@ -0,0 +1,30 @@
+"""
+serializers, that might be helpful for pypeman projects
+"""
+
+import json
+
+
+# attempt to create serializers allowing to encode objects for non
+# Python applications.
+class JsonableEncoder:
+    """ Initial simple encoder for objects.
+        returns either
+        - the object itself if it can be json encoded
+        - a slightly transformed object, that can be json encoded (To be implemented)
+        - a dict containing two fields: type and repr
+    """
+    def encode(self, obj):
+        typ = None
+        try:
+            rslt = json.dumps(obj)
+            typ = "json"
+            return jsonstr
+        except TypeError:
+            pass
+
+        typ = "repr"
```

Comment:
For what `typ` is used for ?

Reply:
a web interface can be adapted if it knows how exactly a message was 'encoded'

typ should provide this information

The major problem is, that pickle does not exist for web interfaces and that the web front end is supposed (lateron) to have a meaningful representation of a message.

It should probably even be customizable for projects.

Reply:
is that OK for you?